src/main/kotlin/org/rust/lang/core/psi/RustLiteral.kt

```diff
@@ -0,0 +1,8 @@
+package org.rust.lang.core.psi
+
+import com.intellij.psi.PsiElement
+import com.intellij.psi.tree.IElementType
+
+interface RustLiteral : PsiElement {
+    val tokenType: IElementType
+}
```

Comment:
Possible things to declare here:
- `val valueString: String` - get a fragment of the source which denotes the value of the literal as-is (without any escaping etc)
- `val suffix: String` - afaik each Rust literal can have a suffix syntactically (not semantically - strings)
- `val value: Any` - ? get literal value as a Java object (problems here: what type for `u64` and `i64`? `long` would be a good candidate, but it supports unsigned integers only in Java 8)